freqtrade/rpc/telegram.py

```diff
@@ -436,9 +436,13 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
             # Notification disabled
             return
 
-        message = self.compose_message(msg, msg_type)
-
-        self._send_msg(message, disable_notification=(noti == 'silent'))
+        # Would this be better than adding un-needed if statements to compose_message?
+        try:
+            message = self.compose_message(msg, msg_type)
+            self._send_msg(message, disable_notification=(noti == 'silent'))
```

Comment:
well that's technically the same as not raising the exception - which would be the better solution :laughing:

Reply:
So it's `compose_message` that raises an error. I think it would break some things if we just outright removed it upon further inspection, so I believe this is the best way to just "skip" it.

Reply:
the only think not raising would break is tests - which can change without problem if behavior changes :)

Reply:
Sounds good :+1: